force-app/main/default/classes/RD2_ETableController.cls

```diff
@@ -303,7 +305,6 @@ public with sharing class RD2_ETableController {
         String nextDonation = '';
 		String nextDonationStringDate = String.valueOf(recurringDonation.npe03__Next_Payment_Date__c);
         if(recurringDonation.RecurringType__c == RD2_Constants.RECURRING_TYPE_FIXED && !String.isEmpty(nextDonationStringDate) ) {
-            //nextDonation = System.Label.RD2_PaidInstallments + ' ' + String.valueOf(recurringDonation.npe03__Total_Paid_Installments__c) +'/'+ String.valueOf(recurringDonation.npe03__Installments__c) + ', ' + System.Label.RD2_NextDonationDate + ' ' + String.valueOf(recurringDonation.npe03__Next_Payment_Date__c);
 			nextDonation = System.Label.RD2_PaidInstallments + ' ' + String.valueOf(recurringDonation.npe03__Total_Paid_Installments__c) +'/'+ String.valueOf(recurringDonation.npe03__Installments__c) + ', ' + System.Label.RD2_NextDonationDate + ' ' + (recurringDonation.npe03__Next_Payment_Date__c).month() + '/'+ (recurringDonation.npe03__Next_Payment_Date__c).day() + '/' + (recurringDonation.npe03__Next_Payment_Date__c).year();
```

Comment:
This hardcodes formatting the date as mm/dd/yyyy. That is a US-centric date format. It should use the users current locale to properly format the date. Typically this is done in the LWC UI using the date component, but you maybe able to format it with date.format(). Testing needed for other locales however.

Reply:
Thank you for the update. Will keep an eye out for that PR.